src/nodes.coffee

```diff
@@ -535,6 +538,17 @@ exports.NaNLiteral = class NaNLiteral extends NumberLiteral
 
 exports.StringLiteral = class StringLiteral extends Literal
 
+  compileNode: (o) ->
+    res = if @csx then [@makeCode @unquote yes] else super()
+
+  unquote: (literal) ->
+    unquoted = @value[1...-1]
+    if literal
+      unquoted.replace /\\n/g, '\n'
+        .replace /\\"/g, '"'
```

Comment:
We format strings to JS strings, which means escaping new lines and quotes. This function removes it, so the string's content is printed out literally.

"b\\"a" throws syntax error, so that can never happen.